src/components/inputs/Radio.test.tsx

```diff
@@ -33,132 +32,136 @@ describe('Radio', () => {
 	});
 
 	describe('Uncontrolled mode', () => {
-		test('should call onChange with the new checked status', () => {
+		test('should call onChange with the new checked status', async () => {
 			const onChangeFn = jest.fn();
-			render(<Radio label={'the label'} onChange={onChangeFn} defaultChecked />);
+			const { user } = setup(<Radio label={'the label'} onChange={onChangeFn} defaultChecked />);
 			const radio = screen.getByRole('radio');
-			// onChange is called on render
+			// onChange is called on setup
```

Comment:
keep `render` here, it is meaningful
```suggestion
			// onChange is called on render
```